gulpfile.js

```diff
@@ -12,6 +12,13 @@ var preprocess = require('gulp-preprocess');
 var preprocessify = require('preprocessify');
 var merge = require('merge-stream');
 
+gulp.task('newcore', function() {
```

Comment:
Basically to use the new core while changing as little as possible, I decided to generate the core folder from the new core lib and then let everything run as it has been doing so before.

Reply:
This seems like a good compromise.